pkg/parser/parser.go

```diff
@@ -148,6 +148,8 @@ func isParam(line string, tool *types.Tool) (_ bool, err error) {
 		}
 	case "credentials", "creds", "credential", "cred":
 		tool.Parameters.Credentials = append(tool.Parameters.Credentials, value)
+	case "exportcredentials", "exportcreds", "exportcredential", "exportcred", "sharecredentials", "sharecreds", "sharecredential", "sharecred":
```

Comment:
don't support the "export" keyword. For new things we are only adding "share" since we want everything to switch to share.

Reply:
fixed

Reply:
Well I pushed another commit at least, but it isn't showing up in the PR 🤔